src/app/modules/choice/choice.component.html

```diff
@@ -8,6 +8,7 @@
     attr.data-auto-id="{{dataAutoId}}" 
     (change)="onChange($event)"
 />
-<label for="{{ value +'-'+ index }}">
-    <span *ngIf="showTitle" class="{{labelClass}}">{{ label ? label : value }}</span>
+<label for="{{ value +'-'+ index }}" class="{{labelClass}}">
+    <span *ngIf="showTitle" >{{ label ? label : value }}</span>
+    <span *ngIf="error" class="ds-c-field__hint ds-u-color--error" id="undefined-message" role="alert">{{ errorMessage }}</span>
```

Comment:
can you name the id something unique? if there's more than one checkbox on the page we'll have duplicate elements with the same id

maybe something like

id="{{ value +'-'+ index }}-error"

Reply:
also, maybe instead of a separate boolean and string variable, you can just use the string to check if the error should show?

Reply:
I thought about that too but that was the way it was done before so I stuck with that, I can update that